File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Eagle.js
const Bird = require('./Bird');
const Cage = require('./Cage'); 

class Eagle extends Bird {

    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    feed(food) {
        throw new Error('Not supported yet.');
    }
    
}

module.exports = Eagle;

File: hw/travez/u2/HW18ZooInPython/ec_edu_espe_zoo_model/Eagle.py
from datetime import datetime
from Bird import Bird


# Ismael Travez, The Java Bandits, DCCO-ESPE

class Eagle(Bird):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        raise NotImplementedError("Not supported yet.")


